lib/dependencies/ImportDependency.js

```diff
@@ -22,20 +23,12 @@ ImportDependency.Template = class ImportDependencyTemplate {
 	apply(dep, source, outputOptions, requestShortener) {
 		const depBlock = dep.block;
 		const promise = DepBlockHelpers.getDepBlockPromise(depBlock, outputOptions, requestShortener, "import()");
-		const comment = this.getOptionalComment(outputOptions.pathinfo, requestShortener.shorten(dep.request));
+		const comment = outputOptions.pathinfo ? Template.toComment(requestShortener.shorten(dep.request)) : "";
 
 		const content = this.getContent(promise, dep, comment);
 		source.replace(depBlock.range[0], depBlock.range[1] - 1, content);
 	}
 
-	getOptionalComment(pathinfo, shortenedRequest) {
```

Comment:
@sokra note that this is a breaking change for plugins using this api (`getOptionalComment`)